cves/CVE-2013-4249.yml

```diff
@@ -110,7 +117,9 @@ discovered:
     The automated, contest, and developer flags can be true, false, or nil.
 
     If there is no evidence as to how this vulnerability was found, then please explain where you looked.
-  answer:
+  answer: Can't find how this vulnerability was found. No comments in the vulnerability-introducing PR 
+  and no comments in the vulnerability. Also looked on https://code.djangoproject.com/search.
+  fix PR 
   automated:
   contest:
   developer:
```

Comment:
The automated, contest and developer fields were not answered. Although you cannot find the methods of discovery you should still respond with nil.

Reply:
Thank you for the reminder